css/style.css

```diff
@@ -0,0 +1,73 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+/* body{
+    display: block;
+} */
+
+#navbar{
+    /* border: 2px solid red; */
+    position: fixed;
+    left: 0;
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    min-width: 250px;
+    overflow: auto;
+    border-right: 2px solid black;
+    background-color: white;
+}
+
+header{
+    padding: 0.5em 0 0.5em 1em;
+}
+
+.nav-link{
+    text-decoration: none;
+    color: black;
+    text-align: left;
+    border-top: 2px solid black;
+    padding: 0.5em 0 0.5em 1em;
+    margin-top: 0.5em;
+}
+
+main{
+    margin-left: 260px;
+}
+
+ul{
+    list-style: outside;
+    padding-left: 3em;
+}
+
+section{
+    padding-top: 1em;
+}
+
+code{
+    display: block;
+    background-color: rgb(202, 202, 202);
+    padding: 0.3em 0.5em 0.3em 1em;
+    margin: 0.5em;
+    white-space: pre-line;
+    max-width: 55em;

```

Comment:
- [ ] Please try to be more professional, polite, clear and specific in your comments and add solid reasons why student don't need to add `max-width`. Kindly look at the point below, which will help you perform code reviews very well but please don't use my example.
  - I would like to suggest you please remove the `max-width` because a `container` with *display: block;* always does occupy the whole width of according to parent container so you don't need to add `max-width` explicitly.

### Your Comment

> Because we set display:block already so we do not need max-width any more. innocent